Tests/SwiftFormatTests/PrettyPrint/CommentTests.swift

```diff
@@ -199,6 +200,127 @@ final class CommentTests: PrettyPrintTestCase {
     assertPrettyPrintEqual(input: input, expected: expected, linelength: 45)
   }
 
+  func testLineCommentsWithCustomLeadingSpaces() {
```

Comment:
I think it would help to split this test case up - it's quite hard to compare the input/expectation at the moment.

Reply:
it's a copy of the previous tests. I can try and split of both?

Reply:
I'm happy to just have the new tests split, feel free to also clean up the copied-from in another PR though.

Reply:
split!

Reply:
Personally I'd split them into functions so the failure is super obvious if it happens. Currently you'd have to map the failed input/output back to one of the pairs by looking over them all (since the failure would always be on the same line). You could also add a line number to each test case, but at that point having separate functions seems easier.

Any preference @allevato? I'm also happy to take as is, it's just tests.